.github/workflows/pypi-manylinux.yml

```diff
@@ -21,19 +21,19 @@ jobs:
       run: |
         python -m pip install --upgrade pip setuptools wheel twine
     - name: Build manylinux Python wheels
-      uses: RalfG/python-wheels-manylinux-build@v0.3-manylinux1_x86_64
+      uses: RalfG/python-wheels-manylinux-build@v0.3.1-manylinux2014_x86_64
       with:
         python-versions: 'cp36-cp36m cp37-cp37m cp38-cp38'
```

Comment:
Is it correct that 36 and 37 have an m but 38 does not?

Reply:
Indeed it is. See https://docs.python.org/3.8/whatsnew/3.8.html#build-and-c-api-changes 

> Default sys.abiflags became an empty string: the m flag for pymalloc became useless (builds with and without pymalloc are ABI compatible) and so has been removed.